### PR TITLE
cloud: rename three products

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

rename three products:
- TiDB Serverless
- TiDB Dedicated
- TiDB Self-Hosted

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/integrate-tidbcloud-with-zapier.md [hunk 1]
```diff
@@ -160,16 +160,16 @@ The following table lists the actions supported by TiDB Cloud App. Note that som
...
-| Create Cluster | Creates a new cluster. Only supports creating a Serverless Tier cluster. | None |
-| Find Database | Finds an existing database. | A Serverless Tier cluster |
-| Create Database | Creates a new database. | A Serverless Tier cluster |
-| Find Table | Finds an existing Table. | A Serverless Tier cluster and a database |
-| Create Table | Creates a new table. | A Serverless Tier cluster and a database |
-| Create Row | Creates a new row. | A Serverless Tier cluster, a database, and a table |
-| Update Row | Updates an existing row. | A Serverless Tier cluster, a database, and a table |
-| Find Row | Finds a row in a table via a lookup column. | A Serverless Tier cluster, a database, and a table |
-| Find Row (Custom Query) | Finds a row in a table via a custom query the you provide. | A Serverless Tier cluster, a database, and a table |
+| Find Cluster | Finds an existing TiDB Serverless or TiDB Dedicated. | None |
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-zapier.md:163]:
```suggestion
| Find Cluster | Finds an existing TiDB Serverless or TiDB Dedicated cluster. | None |
```

File: tidb-cloud/select-cluster-tier.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
 ---
-title: Select Your Cluster Tier
-summary: Learn how to select your cluster tier on TiDB Cloud.
+title: Select Your Cluster Option
```
Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:2]: 感觉这个表述有点奇怪

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-c
...
 
 ## December 27, 2022
 
 **General changes**
 
-- Upgrade the default TiDB version of all [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
+- Upgrade the default TiDB version of all [Serverless Tier](/tidb-cloud/select-cluster-tier.md#tidb-serverless-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
 
-- The point-in-time recovery (PITR) for Dedicated Tier clusters is now in General Availability (GA).
+- The point-in-time recovery (PITR) for TiDB Dedicated clusters is now in General Availability (GA).
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:22]:
```suggestion
- The point-in-time recovery (PITR) for Dedicated Tier clusters is now in General Availability (GA).
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 4]
```diff
@@ -5,13 +5,13 @@ summary: Learn how to migrate data from MySQL-compatible databases hosted in Ama
 
 # Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration
 
-This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL, Amazon Relational Database Service (RDS), or Google Cloud SQL for MySQL) or on-premises to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
+This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL, Amazon Relational Database Service (RDS), or Google Cloud SQL for MySQL) or a self-hosted one to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:8]:
```suggestion
This document describes how to migrate data from a MySQL-compatible database that is self-hosted or hosted on a cloud provider (Amazon Aurora MySQL, Amazon Relational Database Service (RDS), or Google Cloud SQL for MySQL) to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
```

File: tidb-cloud/select-cluster-tier.md (hunk 2 above)
Comment (qiancai) [reply tidb-cloud/select-cluster-tier.md:2]: Choose a Database Type 可以吗

File: tidb-cloud/select-cluster-tier.md [hunk 5]
```diff
@@ -62,18 +62,18 @@ To get the prefix for your cluster, take the following steps:
...
-- Some of TiDB Cloud features are partially supported or not supported on Serverless Tier. See [Serverless Tier Limitations](/tidb-cloud/serverless-tier-limitations.md) for details.
+- TiDB Serverless is currently in beta and does not have uptime SLA guarantee during beta phase. If you use TiDB Serverless beta to store a commercial or production dataset, any potential risk associated with the use should be taken on your own, and PingCAP shall not be liable for any damage.
+- Some of TiDB Cloud features are partially supported or not supported on TiDB Serverless. See [TiDB Serverless Limitations](/tidb-cloud/serverless-tier-limitations.md) for details.
 
-## Dedicated Tier
+## TiDB Dedicated
 
-The TiDB Cloud Dedicated Tier is dedicated for production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing).
+TiDB Dedicated is dedicated for production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing).
```
Comment (qiancai) [inline tidb-cloud/select-cluster-tier.md:72]:
```suggestion
TiDB Dedicated is for production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing).
```

File: tidb-cloud/tidb-cloud-glossary.md [hunk 6]
```diff
@@ -29,11 +29,11 @@ TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL ed
...
 
-In addition, TiDB Cloud provides a Chat2Query API for Serverless Tier clusters. After it is enabled, TiDB Cloud will automatically create a system Data App called **Chat2Query** and a Chat2Data endpoint in Data Service. You can call this endpoint to let AI generate and execute SQL statements by providing instructions. For more information, see [Get started with Chat2Query API](/tidb-cloud/use-chat2query-api.md).
+In addition, TiDB Cloud provides a Chat2Query API for TiDB Serverless clusters. After it is enabled, TiDB Cloud will automatically create a system Data App called **Chat2Query** and a Chat2Data endpoint in Data Service. You can call this endpoint to let AI generate and execute SQL statements by providing instructions. For more information, see [Get started with Chat2Query API](/tidb-cloud/use-chat2query-api.md).
 
-### cluster tier
+### cluster option
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-glossary.md:34]: 这是否算一个术语

File: tidb-cloud/select-cluster-tier.md (hunk 2 above)
Comment (ran-huang) [reply tidb-cloud/select-cluster-tier.md:2]: Database Type -> Cluster Type，这样呢

File: tidb-cloud/tidb-cloud-glossary.md (hunk 6 above)
Comment (ran-huang) [reply tidb-cloud/tidb-cloud-glossary.md:34]: 感觉不算，可以删了吧

Comment (ran-huang) [inline tidb-cloud/tidb-cloud-glossary.md:34]:
```suggestion
```

File: tidb-cloud/tidb-cloud-glossary.md [hunk 7]
```diff
@@ -29,11 +29,11 @@ TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL ed
...
 
-Determines the functionality and capacity of your cluster. Different cluster tiers provide different numbers of TiDB, TiKV, and TiFlash nodes in your cluster.
+Determines the functionality and capacity of your cluster. Different cluster options provide different numbers of TiDB, TiKV, and TiFlash nodes in your cluster.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-glossary.md:37]:
```suggestion
```

File: tidb-cloud/select-cluster-tier.md (hunk 2 above)
Comment (qiancai) [reply tidb-cloud/select-cluster-tier.md:2]: 的确，是集群类型。 界面的 Tier 如果不改的话，感觉这里也不用改

Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:2]:
```suggestion
title: Select Your Cluster Tier
```

File: tidb-cloud/select-cluster-tier.md [hunk 8]
```diff
@@ -1,27 +1,27 @@
 ---
-title: Select Your Cluster Tier
-summary: Learn how to select your cluster tier on TiDB Cloud.
+title: Select Your Cluster Option
+summary: Learn how to select your cluster option on TiDB Cloud.
```
Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:3]:
```suggestion
summary: Learn how to select your cluster tier on TiDB Cloud.
```

File: tidb-cloud/select-cluster-tier.md [hunk 9]
```diff
@@ -1,27 +1,27 @@
...
 aliases: ['/tidbcloud/developer-tier-cluster']
 ---
 
-# Select Your Cluster Tier
+# Select Your Cluster Option
```
Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:7]:
```suggestion
# Select Your Cluster Tier
```

File: tidb-cloud/select-cluster-tier.md [hunk 10]
```diff
@@ -1,27 +1,27 @@
...
 
-The cluster tier determines the throughput and performance of your cluster.
+The cluster option determines the throughput and performance of your cluster.
```
Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:9]:
```suggestion
The cluster tier determines the throughput and performance of your cluster.
```

File: tidb-cloud/select-cluster-tier.md [hunk 11]
```diff
@@ -1,27 +1,27 @@
...
 
-TiDB Cloud provides the following two options of cluster tiers. Before creating a cluster, you need to consider which option suits your need better.
+TiDB Cloud provides the following two options. Before creating a cluster, you need to consider which option suits your need better.
```
Comment (ran-huang) [inline tidb-cloud/select-cluster-tier.md:11]:
```suggestion
TiDB Cloud provides the following two options of cluster tiers. Before creating a cluster, you need to consider which option suits your need better.
```

File: tidb-cloud/tidb-cloud-glossary.md [hunk 12]
```diff
@@ -29,11 +29,8 @@ TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL ed
...
 
-In addition, TiDB Cloud provides a Chat2Query API for Serverless Tier clusters. After it is enabled, TiDB Cloud will automatically create a system Data App called **Chat2Query** and a Chat2Data endpoint in Data Service. You can call this endpoint to let AI generate and execute SQL statements by providing instructions. For more information, see [Get started with Chat2Query API](/tidb-cloud/use-chat2query-api.md).
+In addition, TiDB Cloud provides a Chat2Query API for TiDB Serverless clusters. After it is enabled, TiDB Cloud will automatically create a system Data App called **Chat2Query** and a Chat2Data endpoint in Data Service. You can call this endpoint to let AI generate and execute SQL statements by providing instructions. For more information, see [Get started with Chat2Query API](/tidb-cloud/use-chat2query-api.md).
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-glossary.md:33]:
```suggestio
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**